src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs

```diff
@@ -1,12 +1,14 @@
 using System;
 using System.Globalization;
 using System.Linq;
-using Fluid;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using Fluid;
```

Comment:
Sort usings, Fluid after System.Linq

Reply:
Nope, System then Microsoft :)

Reply:
I meant `using Fluid` just after `System.Linq`, not under `using Microsoft.Extensions.Options` ;)

But not so important, i think it's better to follow these rules but it should not block a PR.

Reply:
AFAIK we sort usings as the following:

using System.*
using Microsoft.*

the remain usings

Reply:
Nevermind but i'm using the `.editorconfig` of our repo and that (for usings) only contains

    # Sort using and Import directives with System.* appearing first
    dotnet_sort_system_directives_first                        = true

So when i use the right click `Remove and Sort Usings` command it puts Liquid upper.

Reply:
Yep, that's why I need to sort them myself ;)